test/all/node/test-exports.mjs

```diff
@@ -30,40 +33,44 @@ describe(`[all] Window and SW packages`, function() {
     ...getPackages({type: 'window'}),
   ];
 
-  it(`should have top level files for every export in index.mjs (and vise-versa)`, async function() {
+  it(`should have top a level module for every export in index.mjs (and vise-versa)`, async function() {
     for (const pkg of windowAndBrowserPackages) {
       const packagePath = path.join(__dirname, '..', '..', '..', 'packages', pkg.name);
       const indexFile = path.join(packagePath, 'index.mjs');
-      const moduleExports = await import(indexFile);
+      const indexContents = await fs.readFile(indexFile, 'utf-8');
 
-      const publicFiles = glob.sync('*.mjs', {
-        ignore: ['index.mjs', '_version.mjs', '_types.mjs'],
-        cwd: packagePath,
+      // Use the acorn parser to generate a list of named exports.
+      const namedExports = [];
+      const indexAST = acorn.parse(indexContents, {
+        ecmaVersion: 6,
+        sourceType: 'module',
       });
-
-      // Ensure there's a public export for every top-level file in the package.
-      for (const publicFile of publicFiles) {
-        const expectedExportName = path.basename(publicFile, '.mjs');
-
-        if (!(expectedExportName in moduleExports)) {
-          throw new Error(`Unable to find export '${expectedExportName}' in ${indexFile}`);
+      for (const node of indexAST.body) {
+        if (node.type === 'ExportDefaultDeclaration"') {
```

Comment:
Is there an extra `"` character in your string?

Reply:
Ah, good catch! (In this case it doesn't affect things because we don't have default exports)